packages/e2e-test-utils-playwright/src/metrics/index.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * External dependencies
+ */
+import type { Page } from '@playwright/test';
+import * as lighthouse from 'lighthouse/core/index.cjs';
+
+export class Metrics {
+	constructor( public readonly page: Page, public readonly port: number ) {
```

Comment:
On a related note, it seems that the `port` is only needed for the Lighthouse report, not any of the actual metrics, so that supports my concern above.

I think we should either use individual functions where those parameters can be passed as needed, or we should split this into two classes, also since the methods to return specific metrics and the method to return the Lighthouse report are fundamentally different. A Lighthouse report is not a metric, so I'd advise to move that to a separate file/structure.

Reply:
Sure we can split them, but it still needs to be a class.

Reply:
Sounds good, let's split it into 2 classes then 👍